superset/dashboards/commands/importers/v1/utils.py

```diff
@@ -81,6 +81,15 @@ def update_id_refs(config: Dict[str, Any], chart_ids: Dict[str, int]) -> Dict[st
             for old_id, value in metadata["expanded_slices"].items()
         }
 
+    if "default_filters" in metadata:
```

Comment:
nit: This switch type logic on `update_id_refs` if it grows (or even now) is asking for a small refactor IMO

Reply:
Good point!